soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -125,11 +276,14 @@ def map_phys_mem():
         tot = 1 + int(runx("awk '/HugePages_Total/ {print $2}' /proc/meminfo"))
         os.system(f"echo {tot} > /proc/sys/vm/nr_hugepages")
 
-    hugef = open(HUGEPAGE_FILE, "w+")
+    hugef_name = HUGEPAGE_FILE + str(stream_id)
+    hugef = open(hugef_name, "w+")
     hugef.truncate(HUGEPAGESZ)
     mem = mmap.mmap(hugef.fileno(), HUGEPAGESZ)
+    log.info("type of mem is %s", str(type(mem)))
     global_mmaps.append(mem)
-    os.unlink(HUGEPAGE_FILE)
+    # helpful to sometimes comment out for debugging, can then cat the page
```

Comment:
Not sure that comment is all that helpful.  I mean, yes, that's the way hugetlbfs works, the man page says it better than we can here.  If you want a mode to leave the pages behind for debugging, maybe we should add that to the interface?

Reply:
Might be worth while adding a flag to leave the huge pages, I found it helpful when debugging